channels-and-providers/sms/clickatell.mdx

```diff
@@ -3,7 +3,7 @@ title: 'Clickatell'
 description: 'Learn how to use the Clickatell provider to send sms notifications using Novu'
 ---
 
-You can use the [Clickatell](https://www.clickatell.com/) provider to send SMS messages to your customers using the Novu Platform with a single API to create multi-channel experiences.
+You can use the [Clickatell](https://www.clickatell.com/) provider to send SMS messages to your customers using the Novu Platform.
```

Comment:
You can use the [Clickatell](https://www.clickatell.com/) provider to send SMS messages to your customers using the Novu Platform.

Let's see how to do it:

Reply:
@fadkeabhi you need to update this as well!

Reply:
@sumitsaurabh927 I have changed it already. Please take a look at updated file.

Reply:
Oh, my bad!